### ISSUE TITLE
Extinguisher station sprites function weirdly

### ISSUE THREAD

**Issue by PanmanS** [PeasantFoundry/Mechtrauma#21]
![image](https://user-images.githubusercontent.com/42521948/210304641-00cc096f-35f3-4886-8f2d-a546c4d6cdee.png)
extinguisher + station without tank = empty sprite
extingusiher + station with tank = sprite with both extinguisher and tank

**Comment by PeasantFoundry** [comment]
If memory serves, this is an issue with how the sprites are rendered. With containers you can either hide contained items or display them. If you display them, you cannot define the display location by slot, only in linear order. 

So what's happening is that the first item will always be rendered behind the station and the second item will be rendered Infront of it. If both items are there, it will render the item in slot0 first and the item in slot1 second. When there is an extinguisher but no tank, it's rendering the extinguisher but it's rendering it behind the station. 

<ItemContainer capacity="0" maxstacksize="1" itemsperrow="1" slotsperrow="1" hideitems="false" itempos="51,-175" iteminterval="4,30" containedspritedepths="0.81,0.79" canbeselected="true" msg="ExtinguisherStation">

There are a couple of ways around this but they're rather annoying. The argon station gets around this by using decorative sprites that are status effect driven. And since those status effects are banging away at 60 times per second, I've tried to limit the use of them. 

Long story short, I decided to leave them as is for the time being. Unless someone knows something I've missed.

**Comment by PanmanS** [comment]
truly unfortunate
